elixir/apps/web/test/web/live/settings/identity_providers/google_workspace/edit_test.exs

```diff
@@ -169,10 +169,16 @@ defmodule Web.Live.Settings.IdentityProviders.GoogleWorkspace.EditTest do
         }
       )
 
+    adapter_config =
```

Comment:
The form test helper actually sets the input value to a `map` here, which we don't handle in the form itself.

We need to properly set the form attribute here to mimic how it would be in the browser.

Tested this in the browser and it still works.